### ISSUE TITLE
Trait aliases don't work

### ISSUE THREAD

**Issue by pie-flavor** [intellij-rust/intellij-rust#3360]
## Environment

* **Intellij-Rust plugin version:** v0.2.0.2114-183
* **Rust toolchain version:** rustc 1.33.0-nightly (a2b0f247b 2018-12-30)
* **IDE name and version:** CLion 2018.3.3
* **Operating system:** Windows 10 1809

## Problem description

Trait aliases are not supported syntax in the plugin.

## Steps to reproduce

```rust
#![feature(trait_alias)]
trait StringIterator = Iterator<Item=String>;
```

This reports an error in the IDE but not to `cargo check`.
